server/src/test/java/org/opensearch/search/query/QueryProfilePhaseTests.java

```diff
@@ -338,6 +340,14 @@ public void testMinScoreDisablesCountOptimization() throws Exception {
             assertThat(query.getTimeBreakdown().keySet(), not(empty()));
             assertThat(query.getTimeBreakdown().get("score"), greaterThanOrEqualTo(100L));
             assertThat(query.getTimeBreakdown().get("score_count"), equalTo(1L));
+            if (concurrency > 0) {
```

Comment:
nit: you can use `executor != null` check here instead of adding new variable `concurrency`. Sorry for confusion in my last comment.

Reply:
Got it. It has been addressed now.